tests/dynamic-core.prm

```diff
@@ -0,0 +1,104 @@
+# A simple setup for convection in a quarter of a 2d shell. See the
+# manual for more information.
```

Comment:
can you update the comment here to say that that the specific test is for the dynamic core temperature plugin?

For the test to actually run, you will also have to add at least one of the output files. Here, I think the statistics and screen output file should be sufficient. Do you know how to do that, or do you need help?